wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/CellDescription/MLSMigrationCellDescription.swift

```diff
@@ -41,9 +41,9 @@ final class MLSMigrationCellDescription: ConversationMessageCellDescription {
     let accessibilityIdentifier: String? = nil
     let accessibilityLabel: String?
 
-    var message: WireDataModel.ZMConversationMessage?
-    var delegate: ConversationMessageCellDelegate?
-    var actionController: ConversationMessageActionController?
+    weak var message: WireDataModel.ZMConversationMessage?
+    weak var delegate: ConversationMessageCellDelegate?
+    weak var actionController: ConversationMessageActionController?
```

Comment:
Could this have impact for other cases, e.g. if message is not retained elsewhere?
In my opinion delegates should be always weak (very rare exceptions), but the other two vars I'm not sure.